sql/world/base/add_default_simpleitemconfig.sql

```diff
@@ -0,0 +1,1021 @@
+TRUNCATE TABLE mod_auctionhousebot_simpleitemconfig;
+INSERT INTO mod_auctionhousebot_simpleitemconfig (itemID, numStacks, stackSize)
```

Comment:
Right now this config file has only TBC items since we're running TBC content. In the future, I will add WoTLK items.